### DISCUSSION TITLE
Databases: Improvements

### DISCUSSION THREAD

**Discussion by andrasbacsai** [coollabsio/coolify#3167] [Improvement Requests]
Able to fine-tune compose file before start a db

**Comment by CarefulGuru** [comment]
Adding the ability to run replicasets in a MongoDB database would be huge as well

**Comment by pranavmappoli** [comment]
@CarefulGuru So, currently no way of achieving this? bcz, Prisma wont work without replicaset

**Comment by CarefulGuru** [comment]
I have tried a couple ways to do it but haven't succeeded yet...it would have to be done with a specific compose setup, you wouldn't be able to use the database
